app/src/main/java/org/p2p/wallet/infrastructure/proto/serializer/TokenKeysDataStore.kt

```diff
@@ -0,0 +1,34 @@
+package org.p2p.wallet.infrastructure.proto.serializer
+
+import android.content.Context
+import androidx.datastore.core.Serializer
+import androidx.datastore.dataStore
+import org.p2p.wallet.TokenKeysDataStore
+import java.io.InputStream
+import java.io.OutputStream
+
+private const val STORE_FILE_NAME = "token_keys_store.pb"
+
+class TokenKeysDataStoreSerializer(
```

Comment:
TokenKeysDataStore

Reply:
and what is token keys anyway? it is public + private keys? it's Account / Wallets DataStore then